packages/manager/modules/bm-server-components/src/network/controller.js

```diff
@@ -0,0 +1,70 @@
+import isFunction from 'lodash/isFunction';
+
+export default class BmServerNetworkTileController {
+  /* @ngInject */
+  constructor($http, $translate, $q, coreURLBuilder) {
+    this.$http = $http;
+    this.$translate = $translate;
+    this.$q = $q;
+    this.coreURLBuilder = coreURLBuilder;
+  }
+
+  $onInit() {
+    this.manageIpUrl = this.coreURLBuilder.buildURL(
+      'dedicated',
+      '#/ip?serviceName=:serviceName',
+      {
+        serviceName: this.server.name,
+      },
+    );
+    this.vrackInfos = [];
+    this.loading = true;
+    this.loadVrackInfos()
+      .then((res) => {
+        this.vrackInfos = res;
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  loadVrackInfos() {
+    return this.getVrack(this.server.name)
+      .then((results) => {
+        const promises = results.map((vrack) =>
+          this.$http.get(`apiv6/vrack/${vrack}`).then(({ data }) => ({
```

Comment:
Not possible to use iceberg with this ?

Reply:
iceberg does not return name and description required. It returns different data.